manifests/p/PixPin/PixPin/Beta/1.9.11.4/PixPin.PixPin.Beta.locale.zh-CN.yaml

```
@@ -0,0 +1,16 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: PixPin.PixPin.Beta
+PackageVersion: 1.9.11.4
+PackageLocale: zh-CN
+Publisher: Shenzhen Shendu Tujing Technology Co., Ltd.
+PublisherUrl: https://pixpinapp.com/

```

```suggestion
PublisherUrl: https://pixpin.cn/
```

Their official URL has changed.